west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 3dd7043488c5d965284d0bc13926e633d2919f65
+      revision: pull/839/head
```

Comment:
Update to `48467b3ba546883a564a7536cae8de227e7cfc07`.

Reply:
Updated